lib/scarpe/cats_cradle.rb

```diff
@@ -41,6 +41,7 @@ class CCInstance
     include Shoes::Log
     include Scarpe::Test::EventedAssertions
     include Scarpe::Test::Helpers
+    include Scarpe::Exceptions
```

Comment:
If we included Scarpe::Exceptions in Scarpe::Webview, could we use it in most of these places as well? I feel like it's being included a lot of times...